test/org/sagebionetworks/bridge/services/ScheduledActivityServiceRecurringTest.java

```diff
@@ -97,48 +100,37 @@ public void after() {
         }
     }
     
-    // WARNING: The notes here are not correct due to changes we've made in the way we test the time zone.
-    // The last day MSK seems to fall outside the scope of the tests. This needs to be fixed in later
-    // updates but we're currently addressing the duplication of tasks.
     @Test
     public void retrievalActivitiesAcrossTimeAndTimeZones() throws Exception {
-        int year = DateTime.now().getYear();
-        DateTime targetDateTime = DateTime.parse((year+1)+"-09-23T03:39:57.779+03:00");
-                
-        // Start a couple of days ago when creating the account. enrollment is at a fixed time.
-        DateTimeUtils.setCurrentMillisFixed(targetDateTime.minusDays(2).getMillis());
+        DateTime now = NOW; // we are going to change this in the course of the test.
         
+        // Use DateTimeUtils to create the user two days in the past, then restore the time
+        DateTimeUtils.setCurrentMillisFixed(now.minusDays(2).getMillis());
         testUser = helper.getBuilder(ScheduledActivityServiceRecurringTest.class).build();
         
-        // We start this test in the early morning in Russia, in the future so the new user's
-        // enrollment doesn't screw up the test.
-        DateTimeUtils.setCurrentMillisFixed(targetDateTime.getMillis());
-
-        // These time zones are far apart and for our chosen time, Dave will be teleporting to the 
-        // previous day. Our scheduler must do something rational.
-        DateTimeZone MSK = DateTimeZone.forOffsetHours(3);
-        DateTimeZone PST = DateTimeZone.forOffsetHours(-7);
+        DateTimeUtils.setCurrentMillisSystem();
```

Comment:
Minor: Did you mean current system time or NOW? Can current time of day change the veracity of the test? Or do we believe at this point that the scheduler is stable regardless of time of day?